public/planning/postes_cfg/ajax.infos.php

```diff
@@ -25,7 +25,7 @@
 $id=filter_input(INPUT_GET, "id", FILTER_SANITIZE_NUMBER_INT);
 $CSRFToken = filter_input(INPUT_GET, "CSRFToken", FILTER_SANITIZE_STRING);
 $nombre=filter_input(INPUT_GET, "nombre", FILTER_SANITIZE_NUMBER_INT);
-$nom=filter_input(INPUT_GET, "nom", FILTER_SANITIZE_STRING);
+$nom=filter_input(INPUT_GET, "nom");
```

Comment:
The last parameter has been removed to prevent html entities when adding quotes.